### PR TITLE
fix(radio): [radio] amend demo and API bug of radio

### PR DESCRIPTION
**Pull request by Youyou-smiles**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: examples/sites/demos/pc/app/radio/group-options-composition-api.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <tiny-radio-group v-model="value" :options="options"></tiny-radio-group>
+  <tiny-radio-group v-model="value" type="radio" :options="options"></tiny-radio-group>
```
Comment (shenjunjian) [inline examples/sites/demos/pc/app/radio/group-options-composition-api.vue]: "radio" 是默认值， 为什么这2个示例要显示的写上 type="radio"

File: examples/sites/demos/pc/app/radio/webdoc/radio.js [hunk 2]
```diff
@@ -24,10 +24,10 @@ export default {
...
       'desc': {
         'zh-CN':
-          '<p>通过 <code>options</code> 配置式渲染单选组。 <code>options</code> 的类型为 <a href="#IRadioGroupOptions">IRadioGroupOptions[]</a> 。另外还提供 <code>type</code> 属性，配合 <code>options</code> 属性一起使用，默认值为 <code>radio</code> 。可以配置为 <code>button</code> ，配置后单选组将以按钮的形式展示。</p>\n',
+          '<p>通过 <code>options</code> 配置式渲染单选组。另外还提供 <code>type</code> 属性，配合 <code>options</code> 属性一起使用，默认值为 <code>radio</code> 。可以配置为 <code>button</code> ，配置后单选组将以按钮的形式展示。</p>\n',
         'en-US':
           '<p>Provides the <code>options</code> attribute to render radio groups by configuring object array data. This property eliminates the need to insert a <code>&lt;radio&gt;</code> or <code>&lt;radio-button&gt;</code> tag as a slot in the tag. \n\nThe <code>options</code> object array contains three fields: <code>label</code> , <code>text</code> , and <code>events</code> . \n\n The <code>type</code> attribute is also provided, which is used together with the <code>options</code> attribute. The default value is <code>radio</code> . You can also set this parameter to <code>button</code> . After the configuration, the radio group is displayed as a button. </p>\n'
```
Comment (shenjunjian) [inline examples/sites/demos/pc/app/radio/webdoc/radio.js]: 改中文要同步改英文呀。  另外这段英文跟上面的中文内容并不匹配，你再看看

File: examples/sites/demos/pc/app/radio/group-options-composition-api.vue (hunk 1 above)
Comment (Youyou-smiles) [reply examples/sites/demos/pc/app/radio/group-options-composition-api.vue]: API体现，radio下的api点击跳转至相应demo时，api应体现在demo中。
https://clouddevops.huawei.com/#/product/domains/29793/issues/list

Comment (Youyou-smiles) [reply examples/sites/demos/pc/app/radio/group-options-composition-api.vue]: 修改demo案例，增加区分展示

File: examples/sites/demos/pc/app/radio/webdoc/radio.js (hunk 2 above)
Comment (Youyou-smiles) [reply examples/sites/demos/pc/app/radio/webdoc/radio.js]: 修改英文描述

**Review by gimmyhehe — APPROVED** [review]

**Review by zzcr — APPROVED** [review]

**Pull request merged**